### PR TITLE
Clarify behavior of command

### PR DESCRIPTION
**Pull request by nachoalonsoportillo**

### CODE REVIEW

**Comment by Court72** [comment]
@shsagir 

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by shsagir** [comment]
@nachoalonsoportillo Thanks for contacting us with feedback about our product documentation. Depending on the complexity, it could take from a few business days to a few weeks to look further into your question, issue, or suggestion. We'll contact you if we need further information, and you'll receive an email message each time we "@mention" you in a comment. 

Thanks again for submitting your feedback. Your suggestions help improve our documentation.

Related to #1609

File: data-explorer/kusto/management/create-table-command.md [hunk 1]
```diff
@@ -17,7 +17,9 @@ Requires [Database user permission](./access-control/role-based-access-control.m
 
 `.create` `table` *TableName* ([columnName:columnType], ...)  [`with` `(`[`docstring` `=` *Documentation*] [`,` `folder` `=` *FolderName*] `)`]
 
-If the table already exists, the command will return success.
+If a table with the same (case-sensitive) name already exists in the context database, the command will return success withouth changing the existing table.
+
+That is the case even if the folder or docstring clauses have different values than the ones set in the existing table, or if the schema declared in the command doesn't match the schema of the existing table.
```
Comment (shsagir) [inline data-explorer/kusto/management/create-table-command.md:22]:
```suggestion
> [!NOTE]
> If a table with the same (case-sensitive) name already exists in the context of the database, the command returns success without changing the existing table, even in the following scenarios:
>
> - The specified schema doesn't match the schema of the existing table
> - The `folder` or `docstring` parameters are specified with values different from the ones set in the table
```

**Review by shsagir — APPROVED** [review]

**Comment by shsagir** [comment]
@nachoalonsoportillo Thank you for your contribution. We've submitted a documentation change based on your suggestion, and the updated article should be live by tomorrow.

#sign-off

**Pull request merged**